zookeeper-server/src/main/java-filtered/org/apache/zookeeper/version/VersionInfoMain.java

```diff
@@ -20,6 +20,6 @@
 
 public class VersionInfoMain implements org.apache.zookeeper.version.Info {
     public static void main(String[] args) {
-        System.out.println("Apache ZooKeeper, version ${project.version} ${build.time}");
+        System.out.println("Apache ZooKeeper, version ${project.version} ${project.build.outputDirectory}");
```

Comment:
I think `mvngit.commit.abbrev` and [`mvngit.commit.committer.date`](https://github.com/koraktor/mavanagaiata/blob/dc44108acc9399ecae5c60f8cbb2099f88ed0fef/src/main/java/com/github/koraktor/mavanagaiata/mojo/CommitMojo.java#L72) is more appropriate for use to go. `mvngit.commit.abbrev` gives use "15f29b5-dirty" and "cb22c82" depends on "dirty" working directory or not. It is less verbose than `REVISION_HASH` which contains the full hash. `mvngit.commit.committer.date`  is configurable through [SimpleDataFormat](https://github.com/koraktor/mavanagaiata/blob/8b3ee9c77d8e2a96a21df0789a7b5ca53e82f2c1/src/main/java/com/github/koraktor/mavanagaiata/mojo/AbstractGitMojo.java#L39), I think we can output `UTC` timestamp as before using some template code if demand.

Reply:
don't hesitate to drop my PR here and replace with your approach: yes, it seems quite logical for users